zephyrdoom/src/doom/s_sound.c

```diff
@@ -598,7 +598,7 @@ void S_UpdateSounds(mobj_t *listener)
     sfxinfo_t*        sfx;
     channel_t*        c;
 
-    // printf("NRFD-TODO: S_UpdateSounds\n");
+    return;
```

Comment:
Please add some TODO/FIXME so that we know that we need to get rid of this eventually